bundles/remote_services/rsa_spi/include/remote_proxy.h

```diff
@@ -29,6 +29,7 @@
 
 #include "endpoint_listener.h"
 #include "remote_service_admin.h"
+#include "celix_long_hash_map.h"
```

Comment:
Replacing the hashmap for a service is a backwards incompatible update. I think we should postpone that until Celix 3.0.0

Reply:
OK，Let's merge it until Celix 3.0.0